packages/core/src/config/config.ts

```diff
@@ -624,3 +633,69 @@ function assignExisting<T>(target: Record<string, T>, obj: Record<string, T>) {
     }
   }
 }
+
+export function getMergedConfig(config: Config, entrypointAlias?: string): Config {
+  return entrypointAlias
+    ? {
+        ...config,
+        lint: getMergedLintConfig(config, entrypointAlias),
+        'features.openapi': {
+          ...config['features.openapi'],
+          ...config.apis[entrypointAlias]?.['features.openapi'],
+        },
+        // TODO: merge everything else here
+      }
+    : config;
+}
+
+export function getMergedLintConfig(
+  config: Config,
+  entrypointAlias?: string
+): LintConfig {
+  const localLint = entrypointAlias 
+    ? config.apis[entrypointAlias]?.lint
+    : {};
+  const mergedLint = {
+    ...config.rawConfig.lint,
+    // Preserve external additions to the lint config:
```

Comment:
what are those external additions?

Reply:
The `config.lint` gets modified `loadConfig`. So it contains some additional attributes if I remember correctly.